src/main/java/com/google/growpod/controllers/GardenController.java

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.growpod.controllers;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.growpod.data.Garden;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/** Controller for Garden entities. */
+public class GardenController {
```

Comment:
I believe a more conventional name for this sort of object, one that thinly wraps a db entity, is a Data Access Object or [Dao](https://en.wikipedia.org/wiki/Data_access_object).

This name would then be `GardenDao`

Reply:
Makes sense. Will change.